.travis.yml

```diff
@@ -80,58 +71,67 @@ matrix:
         - source ./venv/bin/activate
         - bash <(curl -s https://codecov.io/bash)
 
-# Travis provides only 2 cores, so don't overdo the parallelism and waste memory.
-before_script:
+
+before_install:
+  - set -e
   - |
-      set -e
-      if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
-        files_changed=$(git diff --name-only $TRAVIS_COMMIT_RANGE)
-      else
-        # Pull requests are slightly complicated because merging the PR commit without
-        # rebasing causes it to retain its old commit date. Meaning in history if any
-        # commits have been made on master that post-date it, they will be accidentally
-        # included in the diff if we use the TRAVIS_COMMIT_RANGE variable.
```

Comment:
I'd still keep this comment (or a shorter version of it) to explain why we didn't use `TRAVIS_COMMIT_RANGE` here.

Reply:
Good point, re-added with a bit different wording.

Do note that I did change the logic used; it now matches what I added to AppVeyor last week since the current logic doesn't actually work as expected.  I *think* it does work this way, but time will tell :)